process.md

```diff
@@ -0,0 +1,52 @@
+# **Process for introducing new guide documents**
+
+1. First, a draft is developed outside this repository among authors. To avoid
+   duplication of effort, authors may open an issue. The title should start
+   with "In progress:".
+
+1. Towards the end of the draft phase, at the point the authors wish to engage
+   wider collaboration, the authors may open a PR with the draft document
+   against the `drafts/` directory. This PR will be merged by repository
+   maintainers without review. At this time the above "In progress:" issue
+   should be closed.
+
+1. The document may live in `drafts/` for as long as the authors
+   wish. Repository maintainers will merge any PRs from authors on these drafts
+   without review.
+
+1. When the authors are ready to submit the draft for formal review, they open
+   a PR to move the draft to the `review/` directory. At this point an RFC
+   announcement is made to the Working Group. This starts a 30-day count
+   down. An issue is created (Title: "RFC for Foo guide closes yyyy-mm-dd")
+   under a new Milestone with the due-date 30 days away.
+
+1. During the review time, comments are taken in the form of issues. The title
+   should start with "RC Foo:", where Foo is the name of the guide.
+
+1. Authors address review comments by making PRs to the guide and closing out
+   the comments.
+
+1. After the 30 day period is over, the guide will be moved from the `review/`
+   to the `published/` directory once all issues are resolved. The guide will
+   be given the version 1.0
+
+## **Process for maintaining published guides**
+
+* Repository maintainers will regularly triage incoming issues on published
+  guides with the tags:
+
+  * Minor: The fix is obvious or clear. Does not need expert review and RFC
+    period. (typo)
+
+  * Major: Improvements to guide that should have area expert consensus.
```

Comment:
do we want to say `expert` or just `broader/community consensus`. It's not clear who should be considered an `expert`

Reply:
Good point, I think we want someone who would be an author. We also don't say who should be allowed to author a guide.

I don't think we should say broader or community. That would imply that we (repo maintainers) would need to seek out that consensus. We want someone to stand up and claim that they have the expertise to author a new major version (same as first version).

Reply:
Maybe "broader/community" is fine here, but we just need to be clear later at this point:

> available authors can take on a major version update

Reply:
maybe we can just say `improvement that require more in-depth expertise`? Not a big deal: I'm just trying to avoid saying expert, to avoid argument about who's an expert: npm team vs ossf team, for example.

Reply:
yeah, that sounds good.